app/views/users/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_tag "/users", method: :post, class: "new-user-form" do %>
+<%= label_tag :name, "Full Name" %>
```

Comment:
Formatting question - indent all label_tag and text_field_tag?

Reply:
I used the auto-indentations and it came out flush like this.